spec/humanity_api_spec.rb

```diff
@@ -0,0 +1,180 @@
+require "spec_helper"
+
+describe HumanityApi do
+  let(:humanity_api) { HumanityApi.new }
+
+  let(:key) { '9c5edef7302e2f272f8de273b1d20927d65b1c49' }
```

Comment:
Can we use a dummy key here? Or use `vcr` to play static responses? Or create an unimportant key?